src/fu-util.c

```diff
@@ -727,6 +732,62 @@ fu_util_report_history_for_uri (FuUtilPrivate *priv,
 				  SOUP_MEMORY_COPY, data, strlen (data));
 	status_code = soup_session_send_message (priv->soup_session, msg);
 	g_debug ("server returned: %s", msg->response_body->data);
```

Comment:
Since you're actually parsing the response now, probably don't need this debug line anymore.

Reply:
I'd be a lot more comfortable leaving it in; it allows us to debug user problems without using gdb.

Reply:
OK